app/controllers/lit/incomming_localizations_controller.rb

```diff
@@ -2,32 +2,26 @@
 
 module Lit
   class IncommingLocalizationsController < ApplicationController
-    before_action :find_source
+    before_action :find_source_and_incomming_localizations
     before_action :find_icomming_localization, only: %i[accept destroy]
 
-    def index
-      @incomming_localizations = @source.incomming_localizations
-    end
+    def index; end
 
     def accept
       @incomming_localization.accept
-      Lit.init.cache.refresh_key @incomming_localization.full_key
       respond_to do |format|
         format.html { finish_request }
         format.js
       end
     end
 
     def accept_all
-      @source.incomming_localizations.each do |li|
-        li.accept
-        Lit.init.cache.refresh_key li.full_key
-      end
+      @incomming_localizations.each(&:accept)
```

Comment:
it used to call `Lit.init.cache.refresh_key li.full_key`, are you sure it's no longer required?

Reply:
In `accept` method there is a `update_cache` method call: `https://github.com/prograils/lit/blob/deleted_localizations_synchronization/app/models/lit/incomming_localization.rb#L32. I suppose it's exactly what we want with `refresh_key`?